types/src/transaction/transaction_v1.rs

```diff
@@ -153,6 +153,31 @@ impl TryFrom<TransactionV1> for TransactionV1Json {
 }
 
 impl TransactionV1 {
+    // ctor
```

Comment:
Needs proper rustdoc

```suggestion
    /// 
```